ext/RMagick/extconf.rb

```diff
@@ -67,15 +66,9 @@ def configure_compile_options
         cc = (ENV['CC'] || config::CONFIG['CC'] || 'gcc').split(' ').first
         exit_failure "No C compiler found in ${ENV['PATH']}. See mkmf.log for details." unless find_executable(cc)
 
-        # ugly way to handle which config tool we're going to use...
-        $with_magick_wand = false
+        magick_package = determine_imagemagick_package
 
-        # Check for Magick-config
-        if find_executable('pkg-config')
```

Comment:
is this check elsewhere?

Reply:
It was moved inside ` def determine_imagemagick_package`